src/lime_qt/main.cpp

```diff
@@ -1070,6 +1072,33 @@ void GMainWindow::ShowUpdaterWidgets() {
 }
 #endif
 
+void GMainWindow::ShowUpdatePopup() {
+    QNetworkAccessManager *manager = new QNetworkAccessManager();
+    QObject::connect(manager, &QNetworkAccessManager::finished, this, [=](QNetworkReply *reply) {
+        if (reply->error()) {
+            QMessageBox::critical(
+            this, tr("Cannot Fetch Github"),
+            tr( " "
+                "Could not fetch Github"));
+                return;
```

Comment:
Connection failures shouldn't cause an error message to appear, as it will create an error every time the emulator is used without an internet connection